modules/homa-lite/src/weights.rs

```diff
@@ -40,6 +40,7 @@
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
+#![allow(clippy::unnecessary_cast)]
```

Comment:
you shouldn't edit machine generated file, modify the template instead
https://github.com/AcalaNetwork/Acala/blob/master/templates/runtime-weight-template.hbs

Reply:
Yep regenerated the weights file using the correct template.